django_server/.env.example

```
@@ -0,0 +1,8 @@
+# Django Settings
+SECRET_KEY=your-secret-key-here
+DEBUG=True
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DEBUG key should go before the SECRET_KEY key

(UnorderedKey)